Front-end/src/components/Cards.js

```diff
@@ -208,7 +208,7 @@ function Cards() {
                 </>
               );
             })
-          : filters == "below"
+          : filters === "below"
```

Comment:
We need to streamline the whole Cards component and need to create a more readable and more precise sorting function to make the code look clean.